packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -478,6 +478,14 @@ export interface IContainerRuntimeOptions {
 	 * are engaged as they become available, without giving legacy clients any chance to fail predictably.
 	 */
 	readonly explicitSchemaControl?: boolean;
+
+	/**
+	 * When this property is set to true, the blob manager will not return local ids for attachment
+	 * blobs that it uploads. It will instead return the id returned by storage after the blob has
+	 * been uploaded. This is added to ensure that clients running versions 1.x and older can read blobs
+	 * created by clients running versions 2.x. 1.x didn't support local ids for attachment blobs.
```

Comment:
That's probably too late, but I think we need to put exact build when new functionality showed up, such that clients who are on faster train (like Look) could make a call when to use new functionality.

Reply:
Do you mean put that build number into comments or somewhere else?